build-system/tasks/size.js

```diff
@@ -33,6 +33,15 @@ var tableOptions = {
   hsep: '   |   ',
 };
 
+/**
+ * Through2 transform function - Tracks the original size and the gzipped size
+ * of the file content in the rows array. Passes the file and/or err to the
+ * callback when finished.
+ * @param {!Array<!Array<string>>} rows array to store content size information
+ * @param {!File} file File to process
+ * @param {Object} enc Encoding (not used)
+ * @param {function(Error, File)} cb Callback function
```

Comment:
@jaboc83 no worries, im no expert (still learning it too), but we do eventually want to turn on closure compiler so would be good to get it correct :+1:  appreciate your patience!